assets/js/editor/menu-item.ts

```diff
@@ -124,7 +124,7 @@ export const markItem = (markType: MarkType<EditorSchema>, options: Partial<Menu
 
 export const linkItem = (markType: MarkType<EditorSchema>): MenuItem => new MenuItem({
 	title: "Maak of verwijder link",
-	icon: {dom: html`<i class="fa fa-link"></i>`},
+	icon: {dom: html`<i class="fas fa-link" aria-hidden="true"></i>`},
```

Comment:
Misschien hier een helper method toevoegen, ik denk dat als er iemand een nieuw icoontje gaat maken in ts dat er dan niet netjes aria-hidden toegevoegd wordt.